<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsViewn" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="angleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(136, 138, 133);</string>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="rotateSlider">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(136, 138, 133);</string>
        </property>
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);
font: 57 14pt &quot;Ubuntu&quot;;
</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionopen"/>
   <addaction name="separator"/>
   <addaction name="actionsave"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="Edit_Toolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);
font: 57 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionZoom_In"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actioncrop"/>
   <addaction name="separator"/>
   <addaction name="actionrotate"/>
   <addaction name="separator"/>
   <addaction name="actionselect"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionabout"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/reset.png</normaloff>:/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/zoomIn.png</normaloff>:/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/zoomOut.png</normaloff>:/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+-</string>
   </property>
  </action>
  <action name="actioncrop">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/crop.png</normaloff>:/crop.png</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
   <property name="toolTip">
    <string>crop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionselect">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/select.png</normaloff>:/select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>select</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/about.png</normaloff>:/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionrotate">
   <property name="icon">
    <iconset resource="res/images.qrc">
     <normaloff>:/rotate.png</normaloff>:/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>rotate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGraphicsViewn</class>
   <extends>QGraphicsView</extends>
   <header location="global">qgraphicsviewn.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/images.qrc"/>
 </resources>
 <connections/>
</ui>
